# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: ipc/glue/BackgroundParentImpl.cpp
# Commit: e4b3f3d8ba5f
# Full Hash: e4b3f3d8ba5fd951bf79a2014dcee109255c865f
# Author: Jan Varga <jvarga@mozilla.com>
# Date: 2022-08-31 21:54:47
# Regressor Bug: 1786465
# File Overlap Count: 1
# Description:
#   Bug 1786465 - Rename POriginPrivateFileSystem to PFileSystemManager; r=dom-storage-reviewers,jesup
#   
#   The child actor will be held by FileSystemManager on the child side and the
#   parent actor already holds FileSystemDataManager, so it makes sense to rename
#   the protocol to PFileSystemManager.
# ==============================================================================

diff -r 3b0da4b4bb83 -r e4b3f3d8ba5f ipc/glue/BackgroundParentImpl.cpp
--- a/ipc/glue/BackgroundParentImpl.cpp	Wed Aug 31 09:34:48 2022 +0000
+++ b/ipc/glue/BackgroundParentImpl.cpp	Wed Aug 31 09:34:48 2022 +0000
@@ -491,7 +491,7 @@
 
 mozilla::ipc::IPCResult BackgroundParentImpl::RecvCreateFileSystemManagerParent(
     const PrincipalInfo& aPrincipalInfo,
-    Endpoint<POriginPrivateFileSystemParent>&& aParentEndpoint,
+    Endpoint<PFileSystemManagerParent>&& aParentEndpoint,
     CreateFileSystemManagerParentResolver&& aResolver) {
   AssertIsInMainProcess();
   AssertIsOnBackgroundThread();